articles/ai-services/openai/includes/use-your-data-go.md

```diff
@@ -36,91 +36,93 @@ ms.date: 08/29/2023
-                       // > but also on the latest information available in the designated data source. This grounding data also helps the model avoid generating responses
-                       // > based on outdated or incorrect information.
-                       //
-                       // Quote from here: https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/use-your-data
-                       Type: to.Ptr(azopenai.AzureChatExtensionTypeAzureCognitiveSearch),
-                       Parameters: azopenai.AzureCognitiveSearchChatExtensionConfiguration{
-                           Endpoint:  &searchEndpoint,
-                           IndexName: &searchIndex,
-                           Key:       &searchAPIKey,
-                       },
-                   },
-               },
-           },
-           Deployment: modelDeploymentID,
-       }, nil)
-
-       if err != nil {
-       //  TODO: Update the following line with your application specific error handling logic
-       log.Fatalf("ERROR: %s", err)
-   }
-
-   // Contains contextual information from your Azure chat completion extensions, configured above in `AzureExtensionsOptions`
-   msgContext := resp.Choices[0].Message.Context
-
-       fmt.Fprintf(os.Stderr, "Extensions Context Role: %s\nExtensions Context (length): %d\n",
-           *msgContext.Messages[0].Role,
-           len(*msgContext.Messages[0].Content))
-
-       fmt.Fprintf(os.Stderr, "ChatRole: %s\nChat content: %s\n",
-           *resp.Choices[0].Message.Role,
-           *resp.Choices[0].Message.Content,
-       )
+   
+   	azureOpenAIKey := os.Getenv("AOAIKey")
+   	modelDeploymentID := os.Getenv("AOAIDeploymentId")
+   
+   	// Ex: "https://<your-azure-openai-host>.openai.azure.com"
+   	azureOpenAIEndpoint := os.Getenv("AOAIEndpoint")
+   
+   	// Azure AI Search configuration
+   	searchIndex := os.Getenv("SearchIndex")
+   	searchEndpoint := os.Getenv("SearchEndpoint")
+   	searchAPIKey := os.Getenv("SearchKey")
+   
+   	if azureOpenAIKey == "" || modelDeploymentID == "" || azureOpenAIEndpoint == "" || searchIndex == "" || searchEndpoint == "" || searchAPIKey == "" {
+   		fmt.Fprintf(os.Stderr, "Skipping example, environment variables missing\n")
+   		return
+   	}
+   
+   	keyCredential := azcore.NewKeyCredential(azureOpenAIKey)
+   
+   	// In Azure OpenAI you must deploy a model before you can use it in your client. For more information
+   	// see here: https://learn.microsoft.com/azure/cognitive-services/openai/how-to/create-resource
+   	client, err := azopenai.NewClientWithKeyCredential(azureOpenAIEndpoint, keyCredential, nil)
+   
+   	if err != nil {
+   		//  TODO: Update the following line with your application specific error handling logic
+   		log.Fatalf("ERROR: %s", err)
+   	}
+   	inScope := true
```

Comment:
Is this configuration required for this example to work:

inScope := true
semanticConfiguration := "default"

When I diff your PR against the latest code sample from the engineering team this is an addition that isn't present in the engineering team's sample. If there is an issue with the engineering team's sample we can update it as well, but if not we would like the docs to match that sample as closely as possible with the exception of things like environment variables where the docs have different values: 

https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/ai/azopenai#example-Client.GetChatCompletions-BringYourOwnDataWithCognitiveSearch